rockspec/apisix-master-0.rockspec

```diff
@@ -52,6 +52,7 @@ dependencies = {
     "lua-resty-kafka = 0.07",
     "lua-resty-logger-socket = 2.0-0",
     "skywalking-nginx-lua-plugin = 1.0-0",
+    "argparse = 0.7.1-1",
```

Comment:
what's the license of `argparse`?

Reply:
MIT License, and it is currently maintained by luarocks community.